### DISCUSSION TITLE
What about a Supabase Shop?

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#1418] [Feature Requests]
hey everyone 👋🏻

I see developers or companies that support Supabase, and personally, I would love to be able to walk around with a beautiful supabase sticker on my computer.

Do you plan to launch a shop for supabase?

*Don't hesitate to up this thread if you think Supabase should have a shop too 👀*

**Comment by AngeloAnolin** [comment]
I actually think this is a good idea so that Supabase can also add more funding into their endeavours.

Imagine, a supabase powered store, with checkouts and payment integration, all powered by Supabase itself.

**SupabaseSwagShop**

- Stickers
- Shirts
- Hoodies
- Coffee Mugs
- Jackets
- Pens
- Supabase Discounted Credits

and a whole lot more.

**Comment by Olyno** [comment]
The shop is coming this month, announced in the last community video: https://youtu.be/qETcl3SUfzU?t=21 🎉 🚀
